doc/living_in_belgium.md

```diff
@@ -0,0 +1,71 @@
+## Transportation

```

Comment:
```suggestion
# Living in Belgium 

## Bank account

## Health care

## Taxes

## Registration at the commune



## Transportation

```

Reply:
Addi previous sub-paragraphs to make clear that not all the text for "Living in Belgium" has been copied